src/commands/install-docker-compose.yml

```diff
@@ -21,97 +21,7 @@ parameters:
-          echo "Selected version of docker-compose is $DOCKER_COMPOSE_VERSION"
-        fi
-
-        # check if docker-compose needs to be installed
-        if command -v docker-compose >> /dev/null 2>&1; then
-          if docker-compose --version | grep "$DOCKER_COMPOSE_VERSION" >> /dev/null 2>&1; then
-            echo "docker-compose $DOCKER_COMPOSE_VERSION is already installed"
-            exit 0
-          else
-            echo "A different version of docker-compose is installed ($(docker-compose --version)); removing it"
-            $SUDO rm -f $(command -v docker-compose)
-          fi
-        fi
-
-        # get binary/shasum download URL for specified version
-        if uname -a | grep Darwin >> /dev/null 2>&1; then
-          PLATFORM=darwin
-          HOMEBREW_NO_AUTO_UPDATE=1 brew install coreutils
-        else
-          PLATFORM=linux
-        fi
-
-        DOCKER_COMPOSE_BASE_URL="https://github.com/docker/compose/releases/download/$DOCKER_COMPOSE_VERSION"
-        DOCKER_COMPOSE_SHASUM_URL="$DOCKER_COMPOSE_BASE_URL/docker-compose-$PLATFORM-x86_64.sha256"
-
-        # download binary and shasum
-        curl -O \
-          --silent --show-error --location --fail --retry 3 \
-          "$DOCKER_COMPOSE_SHASUM_URL"
-
-        FILENAME=$(cat docker-compose-$PLATFORM-x86_64.sha256 | awk '{ print $NF }' | sed 's/^\*//')
-
-        curl -O \
-          --silent --show-error --location --fail --retry 3 \
-          "$DOCKER_COMPOSE_BASE_URL/$FILENAME"
-
-        set +e
-        grep "$FILENAME" docker-compose-$PLATFORM-x86_64.sha256 | sha256sum -c -
-        SHASUM_SUCCESS=$?
-        set -e
-
-        if [[ "$SHASUM_SUCCESS" -ne "0" ]]; then
-          # in case of malformatted checksum file, use checksum in release description
-          DOCKER_COMPOSE_RELEASE_BODY=$(curl \
-            --silent --show-error --location --fail --retry 3 \
-            "https://api.github.com/repos/docker/compose/releases/tags/$DOCKER_COMPOSE_VERSION" | \
-            jq '.body')
-
-          if [[ $(echo $DOCKER_COMPOSE_RELEASE_BODY | \
-            grep -o -e "\`................................................................\` | \`$FILENAME") ]]; then
-
-            SHASUM_STRING=$(echo $DOCKER_COMPOSE_RELEASE_BODY | \
-              grep -o -e "\`................................................................\` | \`$FILENAME" | \
-              sed -E 's/`|\|//g')
-          elif [[ $(echo $DOCKER_COMPOSE_RELEASE_BODY | \
-            grep -o -e "\`$FILENAME\` | \`................................................................") ]]; then
-
-            SHASUM_STRING=$(echo $DOCKER_COMPOSE_RELEASE_BODY | \
-              grep -o -e "\`$FILENAME\` | \`................................................................" | \
-              sed -E 's/`|\|//g')
-          fi
-
-          SHASUM=$(echo "$SHASUM_STRING" | sed -E "s/$FILENAME| //g")
-
-          # verify shasum
-          echo "$SHASUM  $FILENAME" | sha256sum -c
-        fi
```

Comment:
I'm removing these since the release notes don't come with a checksum. Maybe it was originally added to support docker-compose older release model (?).